pallets/liquidity-mining/src/tests/invariants.rs

```diff
@@ -506,6 +506,20 @@ prop_compose! {
 	}
 }
 
+prop_compose! {
+	fn arb_deposit2()(
```

Comment:
Why did you create special deposit for `update_global_farm_invariant_1`  with different `valued_shares` range than other invariant tests?

Reply:
I had some strange failing tests, namely IncorrectValueShares, but now changing back it looks good, so we can get rid of this.